settings.gradle.kts

```diff
@@ -80,14 +69,30 @@ gradleEnterprise {
     }
 }
 
+gitRepositories {
```

Comment:
I ask myself if such definition can be moved to the build.gradle.kts of the api module instead of having it as part of the global module.

Reply:
It has to be in the settings.gradle file because it runs early in gradle init process before the projects are created so that it can add src to the projects. There is only one settings file at root of projects as far as I know.